packages/ui/src/components/List/style.less

```diff
@@ -0,0 +1,32 @@
+@import "antd/lib/list/style/index.less";
```

Comment:
seems like styles are unordered, you can turn on `stylelint` and enable `.less` files check in your IDE. Valid config can be found in monorepo root